Sources/GraphQLLanguage/Generated/GraphQLParser.swift

```diff
@@ -632,7 +632,7 @@ internal class GraphQLParser: Parser {
 	}
 	@discardableResult
 	 internal func selection() throws -> SelectionContext {
-		var _localctx: SelectionContext = SelectionContext(_ctx, getState())
+        let _localctx: SelectionContext = SelectionContext(_ctx, getState())
```

Comment:
Nit: worth fixing indentation since this is temporary?

Reply:
Uh, nice catch. Generated code has weird indentation and Xcode did't like it...